src/rgb.rs

```diff
@@ -131,28 +131,28 @@ impl RGB {
         }
     }
 
-    /// Creates a callback of closure function, which takes a set or RGB values, within a color
-    /// space and viewed as one observer, and returns a new set of RGB values, represeting the
-    /// stimulus in another color space, and using another observer.
-    ///
-    /// This conversion uses the spectral represenations of the primaries through the color space
-    /// `Spectra` function, to create a  transformation matrix.
-    pub fn convert(
-        obs_from: Observer,
-        space_from: RgbSpace,
-        obs: Observer,
-        space: RgbSpace,
-    ) -> Box<dyn Fn(&Vector3<f64>) -> Vector3<f64>> {
-        todo!()
-    }
+    // /// Creates a callback of closure function, which takes a set or RGB values, within a color
+    // /// space and viewed as one observer, and returns a new set of RGB values, represeting the
+    // /// stimulus in another color space, and using another observer.
+    // ///
+    // /// This conversion uses the spectral represenations of the primaries through the color space
+    // /// `Spectra` function, to create a  transformation matrix.
+    // pub fn convert(
+    //     obs_from: Observer,
+    //     space_from: RgbSpace,
+    //     obs: Observer,
+    //     space: RgbSpace,
+    // ) -> Box<dyn Fn(&Vector3<f64>) -> Vector3<f64>> {
+    //     todo!()
```

Comment:
Clippy complained that the return type here was very complex (`Box<dyn Fn(&Vector3<f64>) -> Vector3<f64>>`). And since the function is not even implemented, I decided to suggest just commenting it out.

I usually don't like having old code laying around in comments. But since this looks more like "I'm going to implement this as soon as I have the time", I left them in, since it contains nice documentation and all already!

When implemented later, we'll have to think about the function signature to make it easy to read and understand. I agree with clippy that getting a heap allocated closure as a return value from a method is a bit frightening :)

Reply:
I also think it's better to not have not implemented functions. Nothing in the public API or documentation hinted at it not being implemented.